src/assets/api-calls.tsx

```diff
@@ -0,0 +1,23 @@
+const baseUrl = 'https://hireup-be.herokuapp.com/api/v1'
+
+export const getApplicants = (): Promise<any> => {
+	return fetch(`${baseUrl}/applicants`)
+		.then(response => {
+			if (response.ok) {
+				return response.json()
+			} else {
```

Comment:
Nice job getting the fetch going! We can add some error handling here when the data allows.